tealeaf/TeaLeafIOS.entitlements

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-<dict/>
+<dict>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string></string>
+	</array>
+</dict>
```

Comment:
Instead of this, you can dynamically add it in build.js by reading this file and then passing the key value pairs

```
plist.add(config, {
  "com.apple.security.application-groups": []
});
```